supervisor/misc/hardware.py

```diff
@@ -114,9 +115,8 @@ def usb_devices(self) -> List[Device]:
     def input_devices(self) -> Set[str]:
         """Return all input devices."""
         dev_list: Set[str] = set()
-        for device in self.context.list_devices(subsystem="input"):
-            if "NAME" in device.properties:
-                dev_list.add(device.properties["NAME"].replace('"', "").strip())
+        for device in INPUT_DEVICES.iterdir():
+            dev_list.add(str(device))
```

Comment:
Please get the data from host udev. This part get rewrite soon from hwmon. That I can easy copy over, use the devices from `for device in self.devices`